README.md

```diff
@@ -240,43 +214,32 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/henask12/financial-profile-explorer/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
+If you like this project and find it helpful, please consider showing your support. Here are a few ways you can contribute:
+
+-   Star this repository on GitHub.
+-   Share the project with others who might find it useful.
+-   Contribute to the project by submitting bug reports, feature requests, or pull requests.
 
-If you like this project...
+Your support and feedback are greatly appreciated!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
+I would like to thank the following individuals and resources for their inspiration:
 
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+-   My codding Partner's
+-   Microverse Communtiy
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/henask12/financial-profile-explorer/assets/77069011/fb6717d2-a3da-4397-ba97-b35503610de3)

Please give the author of the design the appropriate credits! You can do it in the Acknowledgements section on your README! 🤓 🔧